components/bigquery-components/tests/test_extract_bq_to_dataset.py

```diff
@@ -0,0 +1,73 @@
+import google.cloud.bigquery  # noqa
+from kfp.v2.dsl import Dataset
+from unittest import mock
+
+import bigquery_components
+
+extract_bq_to_dataset = bigquery_components.extract_bq_to_dataset.python_func
+
+
+def test_extract_bq_to_dataset(tmpdir):
+    with mock.patch("google.cloud.bigquery.client.Client") as mock_client, mock.patch(
+        "google.cloud.bigquery.job.ExtractJobConfig"
+    ) as mock_job_config, mock.patch("google.cloud.bigquery.table.Table") as mock_table:
```

Comment:
```suggestion
@mock.patch("google.cloud.bigquery.client.Client")
@mock.patch("google.cloud.bigquery.job.ExtractJobConfig")
@mock.patch("google.cloud.bigquery.table.Table")
def test_extract_bq_to_dataset(tmpdir, mock_client, mock_job_config, mock_table):
```

Reply:
I find this more readable, thoughts?

Reply:
I agree, I prefer that way but I just wanted to be consistent with what it was already there. I will implement it in all the tests of the project :)